schematic/manifest/generator.py

```
@@ -922,7 +924,8 @@ def sort_manifest_fields(self, manifest_fields, order = "schema"):
                 # reorder manifest fields so that root dependencies are first and follow schema order
                 manifest_fields = sorted(manifest_fields, key = lambda x: dependencies_display_names.index(x) if x in dependencies_display_names else len(manifest_fields) -1)
             else:
-                print("No schema provided! Cannot order based on schema without a specified schema and a schema root attribute.")
+                logger.error("No schema provided. Cannot order based on schema without "
+                            "specified data model and a data model root node.")
```

Raise Exception instead of `logger.error()` message.